src/ui/DatePicker.tsx

```diff
@@ -43,6 +43,18 @@ const DatePicker = ({
       minDate={minDate}
       maxDate={maxDate}
       customInput={<Input id={id} />}
+      css={`
+        .react-datepicker {
+          border-radius: 8px;
```

Comment:
I wonder if we should put this border-radius value in some kind of global variable?
I see 8px is being used in a lot of places? 
Maybe nice if we want to change the value later?